app/components/item_location/item-location-container.tsx

```diff
@@ -0,0 +1,126 @@
+"use client";
+import { TItemLocationsItemScanResponse } from "@/app/api/item-location/useItemLocationsItemScan";
+import useItemLocationsMove from "@/app/api/item-location/useItemLocationsMove";
+import { alertClosedWindow } from "@/app/service/shared/alert-close-window";
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Typography,
+} from "@mui/material";
+import { AxiosError } from "axios";
+import { useEffect, useState } from "react";
+import ScanButton from "../common/button/scan-button";
+import LoadingDialog from "../common/dialog/loading-dialog";
+import ItemList from "./item-list";
+import ItemInfoFetcher from "./item-location-fetcher";
+
+export default function ItemLocationContainer() {
+  const [itemId, setItemId] = useState<number>();
```

Comment:
[nits]これが何のitemIdか分かりにくかったので、
```suggestion
  const [selectedItemId, setSelectedItemId] = useState<number>();
```
とするとより分かりやすいかなと思いました！

Reply:
scanされたitemIdなのでscanedItemIdにしました！